servers/src/main/java/tachyon/master/InodeFile.java

```diff
@@ -54,8 +62,12 @@ static InodeFile loadImage(ImageElement ele) throws IOException {
     final String ufsPath = ele.getString("ufsPath");
     final int dependencyId = ele.getInt("depId");
     final long lastModificationTimeMs = ele.getLong("lastModificationTimeMs");
+    final String username = ele.getString("username");
+    final String groupname = ele.getString("groupname");
+    final short permission = ele.getShort("permission");
 
```

Comment:
According to last meeting notes, We don't need to consider the compatible problem before Tachyon 1.0.0 version. Thanks